packages/sanity/src/core/releases/hooks/utils.ts

```diff
@@ -52,14 +54,14 @@ export function getReleasesPerspective({
   releases,
   excluded,
 }: {
-  selectedPerspective: string | undefined
+  selectedPerspective: ReleaseId | undefined | 'published'
```

Comment:
COULD: call this `selectedPerspectiveName` to keep it consistent with what's used in `usePerspective`

Reply:
Sure!